docs/user-guide/slicing.ipynb

```diff
@@ -6,10 +6,19 @@
    "source": [
     "# Slicing\n",
     "\n",
-    "Data in a [Variable](../generated/scipp.Variable.rst#scipp.Variable) or [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) can be indexed in a similar manner to NumPy and xarray.\n",
+    "Objects in scipp can be sliced in two ways. The general way to do this is by `\"positional indexing\"` using indices that that denote position. A second, more limited approach, is to use `\"location indexing\"` which is based on coordinate values."
```

Comment:
- For readable diffs, we are trying to keep one sentence per line in the docs.
- Do we gain anything by saying that the second approach is limited?
- Where did you find the term "location indexing"?  Xarray seems to call this "label-based indexing". Location indicing is easily confused with position (aren't the two synonyms?)